src/FAudio_internal.h

```diff
@@ -28,19 +28,21 @@
 #include "FAPOBase.h"
 #include <stdarg.h>
 
-#ifdef FAUDIO_UNKNOWN_PLATFORM
+#ifdef FAUDIO_WIN32_PLATFORM
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
 #include <math.h>
 #include <assert.h>
 #include <inttypes.h>
 
+#include <windows.h>
```

Comment:
Seems like a pretty heavy include for the library, is this needed everywhere or just platform_win32.c?

Reply:
Yeah I didn't spend much time on it, it's for `lstrcpynA` which lives in `winbase.h` but including just that header made it fail to build. I'll try harder to figure the correct minimal includes for that.